src/index.html

```diff
@@ -11,7 +11,165 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+
+      <fieldset class="fieldset">
```

Comment:
```suggestion
    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application">
      <fieldset class="fieldset">
```
Don't leave empty lines between parent and child elements